.github/workflows/base_setup/action.yaml

```diff
@@ -0,0 +1,51 @@
+name: 'openpilot env setup'
+
+inputs:
```

Comment:
can you add an extra input that we assert is set? then it's hard to accidentally use this setup action without the retry

Reply:
Done